src/libraries/System.Private.Xml/src/System/Xml/Xsl/Xslt/XslAstAnalyzer.cs

```diff
@@ -1054,6 +1054,9 @@ public XslFlags Variable(string prefix, string name)
                 return XslFlags.None;
             }
 
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "Supressing warning about not having the RequiresUnreferencedCode attribute since xsl Scripts are " +
+                "not supported in .NET Core")]
```

Comment:
I assume this is the dead code that can be removed? Can we add a link to the issue tracking removing it?

Reply:
https://github.com/dotnet/runtime/issues/50308 done.